### PR TITLE
add sonar-scanner-cli

### PR DESCRIPTION
**Pull request by TeamoreA**

### CODE REVIEW

**Comment by victortrac** [comment]
Please take the time to explain why this is needed.

**Comment by TeamoreA** [comment]
> Please take the time to explain why this is needed.

I have added a README

File: sonar-scanner-cli/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# sonna-scanner-cli
+ This image helps us to keep a copy of `sonarsource/sonar-scanner-cli` docker image in our CK library and also add additional packages to it. Currently, we only have `sudo` added.
```
Comment (victortrac) [inline sonar-scanner-cli/README.md:2]: This comment only says *what* you have done, which is self-evident from the code.  The question is *why* you need to add sudo.

**Review by victortrac — CHANGES_REQUESTED** [review]

File: sonar-scanner-cli/README.md (hunk 1 above)
Comment (TeamoreA) [reply sonar-scanner-cli/README.md]: I have added more details as to why we need sudo. Please review.

**Review by victortrac — APPROVED** [review]

**Pull request merged**